Tools/autotest/arducopter.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, binary,
         self.log_name = "ArduCopter"
 
         self.sitl = None
+        self.NOT_ARMABLE_MODES_LIST = ["AUTO", "AUTOTUNE", "BRAKE", "CIRCLE", "FLIP", "LAND", "RTL", "SMART_RTL", "AVOID_ADSB"]  # miss FOLLOW as need parameter change
+        self.NOT_SETTABLE_MODES_LIST = ["FLIP", "AUTOTUNE"]
+        self.GPS_ARMABLE_MODES_LIST = ["DRIFT", "GUIDED", "LOITER", "POSHOLD", "THROW", "STEERING"]  # miss GUIDEDNO_GPS
```

Comment:
Guided NoGPS should be in the list below (which it isn't), I don't understand the comment at end of line.

Reply:
I don't know yet how to make GUIDEDNO_GPS works, as it arm without GPS or position.

Reply:
So does Acro, Stabilize, AltHold... I'm not following your point. Why can't Guided_NoGPS be in the Normal list?

Reply:
yep, I check it and put it in normal list